### ISSUE TITLE
Auth User can view their active Empires on /gate

### ISSUE THREAD

**Issue by foohonpie** [ImperialConflict/imperialconflict.com#15]
## API

authenticate with the following user:

- email: `foo@bar.baz`
- password: `foobarbaz`

Then, with auth token, visit:

`GET /user`

Sample Response:

```
{
    "data": {
        "id": 105254,
        "name": "foobar",
        "email": "foo@bar.baz",
        "email_verified_at": null,
        "created_at": "2020-12-09T18:44:56.000000Z",
        "updated_at": "2020-12-09T18:44:56.000000Z",
        "empires": {
            "active": [
                {
                    "id": 388885,
                    "name": "FoohonPie",
                    "avatar_url": null,
                    "family": {
                        "id": 7143
                    },
                    "galaxy": {
                        "name": "Virgo 2",
                        "description": "The <b>training galaxy</b> where all beginners start their quest for glory. When you have managed to aquire <b>10 planets</b> here, you can play in the other galaxies. Use this training period wisely to stand a better chance later.",
                        "map_image_path": "/images/Galaxies/Virgo 2.overview.jpg"
                    }
                },
                {
                    "id": 389578,
                    "name": "FoohonPie",
                    "avatar_url": null,
                    "family": {
                        "id": 7567
                    },
                    "galaxy": {
                        "name": "Manthano 3",
                        "description": "Manthano is a peaceful galaxy that focuses on economic growth and self-actualization. It is sanctuary for the pacifists and the battle-weary.  Aggression is forbidden, and empires are limited in their size to ensure growth opportunities for newer empires.",
                        "map_image_path": "/images/Galaxies/Manthano 3.overview.jpg"
                    }
                }
            ]
        }
    }
}
```

## Design

We are throwing out the existing design, and replacing each active empire with the following:

<img width="264" alt="Screen Shot 2020-12-09 at 11 22 25 AM" src="https://user-images.githubusercontent.com/674157/101677486-7bc07e80-3a11-11eb-88fe-95d9798f485e.png">

Notes:
* The "Enter" link can be a placeholder for now.
* We can ignore images for now, as we don't yet have a plan for handling image assets in general.
* This view of an empire will likely *not* be a reusable component, so no need to plan for other usages.

---

Please let me know if there are any questions!

**Comment by bpkennedy** [comment]
Cool and the gang.  I have some questions!

- [ ] What to do when description longer than card body height?  Options - text ellipsis with tooltip on hover/touch OR click to expand down body (what happens layout-wise to adjacent cards/cards in row below?
- [ ] Should empire description support support html or should it expect plain text?
- [ ] Is there a max limit of empires that can be shown on the gate?  If more than a max limit exist in the data how should we handle showing the additional ones?
- [ ] Other than clicking to enter - does any other part of the card do anything when you click/touch something?
- [ ] Should there be an empty state?  Like if the user's /gate has no empire cards - should we show an empty cars with a prompt to encourage the user to do/create something?
- [ ] For the user image and the large image - should there be a specific fallback image/graphic/icon if the user doesn't have one?  I already have the SafeImage component available which can fallback to an "Cannot Load Image" div if there's a network error or long delay, but that's not the same as intentionally falling back to a default state.

**Comment by foohonpie** [comment]
> What to do when description longer than card body height? Options - text ellipsis with tooltip on hover/touch OR click to expand down body (what happens layout-wise to adjacent cards/cards in row below)?

text ellipsis with tooltip on sounds good.

> Should empire description support html or should it expect plain text?

Plain text.  I'll clean up the data and remove the html.

> Is there a max limit of empires that can be shown on the gate? If more than a max limit exist in the data how should we handle showing the additional ones?

No max needed for now.

> Other than clicking to enter - does any other part of the card do anything when you click/touch something?

If viable, let's make it all clickable to the same place.  Specific link TBD, but we should just have the single destination.

> Should there be an empty state? Like if the user's /gate has no empire cards - should we show an empty cars with a prompt to encourage the user to do/create something?

Yes, but that will come in a different ticket for the joinable galaxies.  Can leave empty state empty for now.

> For the user image and the large image - should there be a specific fallback image/graphic/icon if the user doesn't have one? I already have the SafeImage component available which can fallback to an "Cannot Load Image" div if there's a network error or long delay, but that's not the same as intentionally falling back to a default state.

For both of those, if an image path value is null, let's not render the image.  So the top title section would all shift left, and the main card image would just collapse.

> Do you want me to also make those top and bottom panels with the character pictures and dialogue?

Nah, we can ditch those.

---

Thanks!

**Comment by foohonpie** [comment]
Closed by #18.
